setup.py

```diff
@@ -429,6 +429,19 @@ def macos_extensions():
 
 
 if __name__ == "__main__":
+
+    msg = ("SWIG is a required build dependency of Enable. Furthermore, there "
+          "is currently a known issue with SWIG 4.0, see enthought/enable#360."
+          " Please install SWIG 3.0.x (see http://www.swig.org/).")
+    try:
+        with subprocess.Popen(["swig", "-version"], stdout=subprocess.PIPE, encoding='utf-8') as proc:
+            swig_version_line = proc.stdout.read().split('\n')[1]
+            # We expect text along the lines of "SWIG Version 3.0.11"
+            swig_version = swig_version_line.split(' ')[2]
```

Comment:
Did you consider using a regex here?

Reply:
>  The current code is a little bit hacky looking, although I'm not sure a regex would look much cleaner 🤔

I think it would look cleaner. Instead of splitting lines and then checking that `3` exists in the specific line, you can just use the regex on the stdout directly.